cli/README.md

```diff
@@ -34,7 +34,15 @@ If you have Rust tooling installed on your machine (see
 source code.
 
 In addition to the Rust build toolchain, you also need Go installed. See
-[Go Downloads](https://go.dev/dl/).
+[Go Downloads](https://go.dev/dl/). The minimum required version of Go is 1.22.
+
+**Rust version requirement:** Zinnia requires Rust toolchain version 1.77. If you have `rustup`
```

Comment:
Let's move this block after the first paragraph (which explains Rust tooling) and before the second paragraph describing Go tooling.

At the high-level:
1. Rust toolchain requirements
2. Go toolchain requirements
3. Gcc requirements on Windows

Reply:
Great idea!

Reply:
Resolved in` https://github.com/CheckerNetwork/zinnia/pull/694/commits/e10a2817099a5122af9271579bd1b98809d92f51